hack/generate_swagger_spec.sh

```diff
@@ -21,5 +21,4 @@ echo "+ Clean up go mod"
 go mod tidy
 
 echo "+ Generate swagger spec"
-swag init -g cmd/chaos-server/main.go
-cp doc/swagger.yaml ui/packages/dashboard_client/
+swag init --output ./doc -g cmd/chaos-server/main.go
```

Comment:
Let's open a new good first issue about providing a script for installing this tool after this pr merged.

Also, update Makefile later.